style.css

```
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+.container {
+  display: flex;
+  justify-content: space-around;
+  width: 375px;
+  height: 48 px;
+}
```

- [ OPTIONAL ] Currently, the logo is not correctly aligned with the menu icon compared to the original design.

To fix that, you only need to add `align-items: center;` to the `container` class.
```suggestion
.container {
  display: flex;
  justify-content: space-around;
  align-items: center;
  width: 375px;
  height: 48 px;
}
```